androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -14,15 +18,15 @@ import androidx.media3.ui.PlayerView
 
 private const val MAIN_PACKAGE_PATH_PREFIX = "android.resource://"
 
-@androidx.annotation.OptIn(UnstableApi::class)
 @Composable
 fun EnhancedVideoPlayer(
     resourceId: Int,
     useControls: Boolean = false,
```

Comment:
Does it still make sense to have `useControls`? We can either use the default controls (fallback that is inside your `data class ControlsDecoration` or let the `controlsDecoration` argument (or some other name) receive a `PlayerControls` from whoever is calling